.github/workflows/min-deps.yml

```diff
@@ -0,0 +1,60 @@
+name: min-deps
```

Comment:
There was probably a better way to do this than just making an entire extra workflow file but my github-actions-fu is weak